plone/app/discussion/browser/moderation.pt

```diff
@@ -103,7 +103,12 @@
                                     <input type="hidden" name="selected_obj_paths:list" value="#"
                                            tal:attributes="value item/getURL" />
                                 </td>
-                                <td tal:content="python:item.author_name or item.Creator" />
+                                <td>
+                                    <span tal:content="python:item.author_name or item.Creator">Name</span>
+                                    <div tal:condition="item/author_email"><a 
```

Comment:
One more nitpick from me (sorry!), would you mind adding the ``<a`` on its own line? I did not notice it at the beginning and thought that everything was inside the div.